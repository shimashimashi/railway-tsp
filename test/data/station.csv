station_cd,station_g_cd,station_name,station_name_k,station_name_r,line_cd,pref_cd,post,address,lon,lat,open_ymd,close_ymd,e_status,e_sort
1,1130101,A,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.726413,41.773709,1902-12-10,0000-00-00,0,1110101
1130101,1130101,B,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.726413,41.773709,1902-12-10,0000-00-00,0,1110101
3,3,C,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.733539,41.803557,1902-12-10,0000-00-00,0,1110101
4,4,D,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.722952,41.846457,1902-12-10,0000-00-00,0,1110101
5,5,E,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.71358,41.86464,1902-12-10,0000-00-00,0,1110101
6,9992701,F,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.688556,41.886971,1902-12-10,0000-00-00,0,1110101
7,7,G,,,11101,1,040-0063,北海道函館市若松町１２-１３,140.646525,41.9054,1902-12-10,0000-00-00,0,1110101