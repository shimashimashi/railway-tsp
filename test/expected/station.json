{"stations":[{"address":"北海道函館市若松町１２-１３","lat":41.773709,"line_code":11101,"lon":140.726413,"post":"040-0063","prefecture":"北海道","station_code":1,"station_group_code":1130101,"station_name":"A"},{"address":"北海道函館市若松町１２-１３","lat":41.773709,"line_code":11101,"lon":140.726413,"post":"040-0063","prefecture":"北海道","station_code":1130101,"station_group_code":1130101,"station_name":"B"},{"address":"北海道函館市若松町１２-１３","lat":41.803557,"line_code":11101,"lon":140.733539,"post":"040-0063","prefecture":"北海道","station_code":3,"station_group_code":3,"station_name":"C"},{"address":"北海道函館市若松町１２-１３","lat":41.846457,"line_code":11101,"lon":140.722952,"post":"040-0063","prefecture":"北海道","station_code":4,"station_group_code":4,"station_name":"D"},{"address":"北海道函館市若松町１２-１３","lat":41.86464,"line_code":11101,"lon":140.71358,"post":"040-0063","prefecture":"北海道","station_code":5,"station_group_code":5,"station_name":"E"},{"address":"北海道函館市若松町１２-１３","lat":41.886971,"line_code":11101,"lon":140.688556,"post":"040-0063","prefecture":"北海道","station_code":6,"station_group_code":9992701,"station_name":"F"},{"address":"北海道函館市若松町１２-１３","lat":41.9054,"line_code":11101,"lon":140.646525,"post":"040-0063","prefecture":"北海道","station_code":7,"station_group_code":7,"station_name":"G"}]}
